projects/lib-shared-modules/src/lib/components/filter/filter.component.html

```diff
@@ -2,7 +2,7 @@
   <div class="container">
     @for(filter of filterData;track filter){
       <div class="filter-container">
-            <mat-select [placeholder]="filter.label" >
+            <mat-select [placeholder]="filter.label" multiple>
```

Comment:
when multiple options are selected all those options are displayed in the label-box .
other than this any changes please, let me know

Reply:
i get that, but how would you configure this is a multi select filter or single select filter, because i can see just "multiple" attribute only passed in here.